minecraft_servers/combatreforged/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "server_name": "combatreforged",
+  "nice_name": "CombatReforged",
+  "direct_ip": "combatreforged.com",
+  "server_wildcards": [
+    "%.combatreforged.com"
+  ],
+  "supported_languages": [
+     "en"
+  ],
+  "social": {
+    "web": "https://venade.net",
```

Comment:
I do not understand the website content:

<img width="269" alt="Bildschirm­foto 2023-02-27 um 18 48 14" src="https://user-images.githubusercontent.com/45363287/221642952-2f3008d3-6c4f-41f9-b14a-7a84a4598bc4.png">

Reply:
Currently there is no Website. I can remove it from the manifest.json for time being.